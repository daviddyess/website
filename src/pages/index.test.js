import React from 'react';
import renderer from 'react-test-renderer';

import IndexPage from './index';

describe('PrivacyPage', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<IndexPage />).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
