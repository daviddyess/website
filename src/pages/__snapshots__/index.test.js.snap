// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PrivacyPage renders correctly 1`] = `
Array [
  <nav
    className="mb-5 navbar navbar-expand-lg navbar-dark bg-dark"
  >
    <a
      className="navbar-brand"
      href="/"
    />
    <button
      aria-controls="basic-navbar-nav"
      aria-label="Toggle navigation"
      className="navbar-toggler"
      onClick={[Function]}
      type="button"
    >
      <span
        className="navbar-toggler-icon"
      />
    </button>
    <div
      aria-expanded={null}
      className="navbar-collapse collapse"
      id="basic-navbar-nav"
    >
      <div
        className="mr-auto navbar-nav"
        onKeyDown={[Function]}
      >
        <div
          className="nav-item"
        >
          <a
            className="nav-link"
            data-rb-event-key={null}
            disabled={false}
            href="/"
            onClick={[Function]}
          >
            Home
          </a>
        </div>
        <a
          className="nav-link"
          data-rb-event-key={null}
          disabled={false}
          href="/about"
          onClick={[Function]}
        >
          About Us
        </a>
        <a
          className="nav-link"
          data-rb-event-key={null}
          disabled={false}
          href="/privacy"
          onClick={[Function]}
        >
          Privacy Policy
        </a>
        <div
          className="nav-item"
        >
          <a
            className="nav-link"
            data-rb-event-key="https://github.com/gusta-project"
            href="https://github.com/gusta-project"
            onClick={[Function]}
            onKeyDown={[Function]}
            rel="noopener noreferrer"
            target="_blank"
          >
            GitHub
          </a>
        </div>
      </div>
    </div>
  </nav>,
  <div
    className="container"
  >
    <main>
      <div
        className="container"
      >
        <div
          className="row"
        >
          <div
            className="col"
          >
            <div
              className="row"
            >
              <h1
                className="gp-heading gp-heading--primary"
              >
                Gusta Project
              </h1>
            </div>
            <div
              className="row"
            >
              <p>
                This project aims to provide a free and open experience for mixing e-liquid.
              </p>
            </div>
          </div>
        </div>
      </div>
    </main>
    <footer
      className="mt-5 text-right"
    >
      <p>
        Â© 
        2019
         
        Company
        . Released under the MIT License.
      </p>
    </footer>
  </div>,
]
`;
